lib/active_model/serializer/associations.rb

```diff
@@ -13,9 +13,8 @@ module Associations
       DEFAULT_INCLUDE_TREE = ActiveModel::Serializer::IncludeTree.from_string('*')
 
       included do |base|
-        class << base
-          attr_accessor :_reflections
-        end
+        base.class_attribute :_reflections
```

Comment:
did you intend to allow instance reader/writers? did you intend to make it inheritable?